conv_rnn/README.md

```diff
@@ -0,0 +1,18 @@
+## Convolutional RNN
+
+Implementation based on [[1]](http://dl.acm.org/citation.cfm?id=3098140).
+
+### Usage
+
+Run `./getData.sh` to fetch the data. The project structure should now look like this:
+
+```
+├── conv_rnn/
+│   ├── data/
```

Comment:
Please move all data to https://github.com/castorini/data?

Reply:
Oh I used the script from https://github.com/castorini/Castor/tree/master/kim_cnn